modules/core/src/lib/layers/editable-geojson-layer.js

```diff
@@ -94,11 +94,11 @@ type State = {
 };
 
 export default class EditableGeoJsonLayer extends EditableLayer {
-  state: State;
+  // state: State;
 
-  props: Props;
+  // props: Props;
 
-  setState: ($Shape<State>) => void;
+  // setState: ($Shape<State>) => void;
```

Comment:
@supersonicclay Why we needed these?

Reply:
I didn't spend much time on it, but it was to add extra flow coverage. Not sure why it didn't "just work" with deck.gl's flow annotations.